db-server/test/backend/remote.js

```diff
@@ -1572,13 +1576,20 @@ module.exports = function(cfg, makeServer) {
             })
             .then(function (r) {
               respOk(r)
-              return client.getThen('/sessionToken/' + user.sessionTokenId + '/verified')
+              return P.all([
+                client.getThen('/sessionToken/' + user.sessionTokenId + '/verified'),
+                client.getThen('/keyFetchToken/' + user.keyFetchTokenId + '/verified'),
+              ])
             })
             .then(function (r) {
-              respOk(r)
-              const result = r.obj
-              assert.equal(result.tokenVerificationCodeHash, null, 'tokenVerificationCodeHash not set')
-              assert.equal(result.tokenVerificationCodeExpiresAt, null, 'tokenVerificationCodeExpiresAt not set')
+              respOk(r[0])
+              respOk(r[1])
+              const sessionToken = r[0].obj
+              const keyFetchToken = r[1].obj
```

Comment:
No biggie, but you could switch to a `.spread` here and above so that you don't need to index into the result array and get the two results passed in with appropriate names instead.